recipes/ckan/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set name = "ckan" %}
+{% set version = "2.8.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/ckan/{{ name }}/archive/{{ name }}-{{ version }}.tar.gz
+  sha256: 1fdc09f4b1ae3a70d84138118c86eee056695235e5be8c42501c8af6b6679686
+
+build:
+  number: 0
+  entry_points:
+    - ckan = ckan.cli.cli:ckan
+  script: "{{ PYTHON }} -m pip install . -vv "
+  skip: True  # [win]
+
+requirements:
+  host:
+    - pip
+    - python <3.0
+    - setuptools
+  run:
+    - python <3.0
+    - alembic ==1.0.0
+    - babel ==2.3.4
+    - bleach ==3.0.2
+    - click ==6.7
+    - fanstatic >=0.12
+    - flask ==1.1.1
+    - flask-babel ==0.11.2
+    - jinja2 ==2.10.1
+    - markdown ==2.6.7
+    - passlib >=1.6.5
+    - paste >=1.7.5.1
+    - PasteScript ==2.0.2
+    - polib ==1.0.7
+    - psycopg2 ==2.8.2
+    - python-magic ==0.4.15  # not available for WIN on conda-forge, as well as libmagic
+    - pysolr ==3.6.0
+    - pylons >=0.9.7
+    - python-dateutil >=1.5.0
+    - pytz ==2016.7
+    - pyutilib ==5.7.1
+    - pyyaml  # needed by webassets. latest should be fine.
+    - repoze.who-friendlyform ==1.0.8
+    - repoze.who ==2.3
+    - requests ==2.22.0
+    - routes ==1.13
+    - rq ==1.0
+    - simplejson >=3.10.0
+    - sqlalchemy-migrate ==0.12.0
+    - sqlalchemy ==1.3.5
+    - sqlparse ==0.2.2
+    - tzlocal ==1.3
+    - unicodecsv >=0.9
+    - vdm ==0.15
+    - webassets ==0.12.1
+    - webhelpers ==1.3
+    - webob >=1.0.8
+    - webtest >=1.4.3  # need to pin this so that Pylons does not install a newer version that conflicts with WebOb==1.0.8
+    - werkzeug ==0.15.5
+    - zope.interface >=4.3.2
+
+test:
+  imports:
+    - ckan
```

Comment:
Would it be possible to add other tests to CKAN? We should at least run the ckan command.  Additionally, if unit-tests are shipped with the src package it would be super helpful because of the number of dependencies in this package.

Reply:
@sodre the ckan.plugins test has been added as well as the ckan paster test commands